content/en/docs/instrumentation/java/exporters.md

```diff
@@ -171,14 +219,240 @@ public class DiceApplication {
 }
 ```
 
-To see the traces exported quickly, you can run Jaeger with OTLP enabled in a
-docker container:
+## Console
+
+To debug your instrumentation or see the values locally in development, you can
+use exporters writing telemetry data to the console (stdout).
+
+If you followed the
+[Getting Started](/docs/instrumentation/java/getting-started/) or
+[Manual Instrumentation](/docs/instrumentation/java/manual/) guides, you already
+have the console exporter installed.
+
+The `LoggingSpanExporter`, the `LoggingMetricExporter` and the
+`SystemOutLogRecordExporter` are included in the
+`opentelemetry-exporter-logging` artifact.
+
+## Jaeger
+
+[Jaeger](https://www.jaegertracing.io/) natively supports OTLP to receive trace
+data. You can run Jaeger in a docker container with the UI accessible on port
+16686 and OTLP enabled on ports 4137 and 4138:
 
 ```shell
-docker run -d --name jaeger \
-  -e COLLECTOR_OTLP_ENABLED=true \
+docker run --rm \
+  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
   -p 16686:16686 \
   -p 4317:4317 \
   -p 4318:4318 \
+  -p 9411:9411 \
   jaegertracing/all-in-one:latest
 ```
+
+Now following the instruction to setup the [OTLP exporters](#otlp-dependencies).
+
+## Prometheus
+
+To send your metric data to [Prometheus](https://prometheus.io/), you can either
+[enable Prometheus' OTLP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
+and use the [OTLP exporter](#otlp) or you can use the `PrometheusExporter`.
```

Comment:
The camel case of "PrometheusExporter" suggests its the name of a class in java. We call it the "prometheus exporter", but the class name is `PrometheusHttpServer`. Not sure how you want to reflect that in this document.

Reply:
```suggestion
and use the [OTLP exporter](#otlp) or you can use the [`PrometheusHttpServer`](https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-prometheus/latest/io/opentelemetry/exporter/prometheus/PrometheusHttpServer.html), a `MetricReader`, that starts an HTTP server that will collect metrics and serialize to Prometheus text format on request.
```

Is it OK to just replace it like this? I used the wording from the API docs.

Reply:
Sorry for the delay. Yeah that looks good.